src/Config/default.ts

```diff
@@ -1,19 +1,14 @@
 import * as dotenv from 'dotenv';
```

Comment:
do you still using dotenv? if the answer is no, please remove it unnecessary imports.

Reply:
Yes,  I still using dotenv, but the package @type/dotenv is deprecated. So i uninstalled it.